pkg/mcp/configz/configz_test.go

```diff
@@ -131,28 +129,14 @@ func testConfigZWithNoRequest(t *testing.T, baseURL string) {
 	if !strings.Contains(data, testEmptyCollection) {
 		t.Fatalf("Collections should have been displayed: %q", data)
 	}
-	want := 2
+	want := 1
 	if got := strings.Count(data, testEmptyCollection); got != want {
-		t.Fatalf("Only the collection and initial ACK request should have been displayed: got %v want %v: %q",
+		t.Fatalf("Only the collection should have been displayed: got %v want %v: %q",
 			got, want, data)
 	}
 }
 
-func testConfigZWithOneRequest(t *testing.T, baseURL string) {
-	t.Helper()
-
-	for i := 0; i < 10; i++ {
-		data := request(t, baseURL+"/configz")
-		if strings.Count(data, testEmptyCollection) != 3 {
-			time.Sleep(time.Millisecond * 100)
-			continue
-		}
-		return
-	}
-	t.Fatal("Both collections, the initial request, and a recent ACK request should have been displayed")
-}
-
-func testConfigJWithOneRequest(t *testing.T, baseURL string) {
+func testConfigJWithTwoRequests(t *testing.T, baseURL string) {
```

Comment:
This looks like only one request is being used in the test.  Am I missing something?

Reply:
It is about having two "LatestRequests" in the list. I've updated the test name to clarify.
The  LatestRequest check is not visible in the diff. You can see it by expanding the diff output.